Improved UI: Shadows, buttons, table, card width

fredoist opened the pull request:
This PR adds improvements for a more natural and clean UI. Also fixed some bugs.
Related to #5

fredoist committed:
Improved card shadows, added more natural shadows

fredoist committed:
Fixed linting errors

fredoist committed:
Improve apply button padding

fredoist committed:
Remove card borders and adjusted mobile width

fredoist committed:
Fixed background URLs

fredoist committed:
Improved table appearance

fredoist committed:
Added hover transitions

iampavangandhi commented:
@agonzalezio Please provide screenshots of what you did comparing before and after :smile:

fredoist commented:
You can preview the changes in the deploy preview: https://deploy-preview-7--hackrcdu.netlify.app/

Basically: added more whitespace between sections, added more natural shadows to cards (the old ones were too darken), added hover transitions on button and theme cards, adjusted some widths to be responsive on mobile, and removed some unused CSS.

BTW, when designing for mobile you should only add the changing properties inside the `@media` elements, when you copy the entire desktop properties and modify or add some for mobile, it's more dificult to edit and maintain, because you must edit the same styles two times; ej. for changing card's with or background.

iampavangandhi commented:
@agonzalezio Thank you for your contributions 👍🏻
